project.yml

```
@@ -30,8 +30,8 @@ targets:
     sources: [FlashSpace]
     settings:
       base:
-        MARKETING_VERSION: 2.7.34
-        CURRENT_PROJECT_VERSION: 34
+        MARKETING_VERSION: 2.7.35
```

```suggestion
        MARKETING_VERSION: 2.8.35
```

I increase minor when new feature is added :)